tests/test_utils_platform_specific.py

```diff
@@ -40,7 +46,3 @@ def test_start_script_and_exit(self):
         # ensure file has been moved from src to dst
         self.assertTrue(any(dst_dir.iterdir()))
         self.assertTrue((dst_dir / src_file_name).exists())
-        # original src file no longer exists
```

Comment:
This should remain in place.

Reply:
Can you explain why? We shouldn't have different behaviour for Mac and Windows, for just removing the src dir

Reply:
Indeed we should not have different behavior. That's why I wondered why you removed the assertions. These assertions test desired behavior. If the mac implementation does not comply, you should fix the implementation instead of removing the assertions.

Reply:
I did originally, but you said in a previous comment, that we did not want to remove the src directory. Basically, it was passing on windows, but not on Mac. Can you clarify if you want the src dir kept or gone, and what you want the test case to be checking exactly?

Reply:
yes and they are still present in the test case

```
         self.assertTrue(any(dst_dir.iterdir()))
         self.assertTrue((dst_dir / src_file_name).exists())
```

Reply:
The comments pertain to line 43-46, in red, which have been removed.

Reply:
Okay. They have now been restored. As you can see, the MacOS tests, fails.
What do  you suggest?

Reply:
And that wont stop notsotuf from performing patch updates?

Reply:
aha fair enough.
We still have the issue of the MacOS tests not passing.

`self.assertFalse(src_file_path.exists())`

But you said you didn't have an issue with the src dir being deleted right?
So do you want me to fix that?

Reply:
Yes, that would be great. :-)